File: Latihan_A/Pc.cpp
class Pc
{
    private:
        // private atribut
        Processor objProc;
        Disk objStorage;
        Ram objMemory;
        int totalPrice;

    public:
        // constructor
        Pc(){}
        Pc(Processor objProc, Disk objStorage, Ram objMemory, int totalPrice) {
            this->objProc = objProc;
            this->objStorage = objStorage;
            this->objMemory = objMemory;
            this->totalPrice = totalPrice;
        }

        // method get dan set untuk objProc, objStorage, objMemory, dan totalPrice
        
        void setProcie(Processor objProc) {
            this->objProc = objProc;
        }
        void setStorage(Disk objStorage) {
            this->objStorage = objStorage;
        }
        void setMemory(Ram objMemory) {
            this->objMemory = objMemory;
        }
        void setTotalPrice(int totalPrice) {
            this->totalPrice = totalPrice;
        }
        
        Processor getProcie() {
            return this->objProc;
        }
        Disk getStorage() {
            return this->objStorage;
        }
        Ram getMemory() {
            return this->objMemory;
        }
        int getTotalPrice() {
            return this->totalPrice;
        }

        // Print method
        void printComputer(){
            cout << "BAHASA C++\n";
            this->objProc.printProcie();
            this->objStorage.printDisk();
            this->objMemory.printRam();
            cout << "Total Price : " << "Rp. " << this->totalPrice << endl;
        }

        // destructor
        ~Pc(){}
};

File: Latihan_B/php/Pc.php
<?php
include "Processor.php";
include "Disk.php";
include "Ram.php";

class Pc {
    private $procie;
    private $storage;
    private $memory;
    private $totalPrice;

    // constructor
    function __construct($procie, $storage, $memory, $totalPrice) {
        $this->procie = $procie;
        $this->storage = $storage;
        $this->memory = $memory;
        $this->totalPrice = $totalPrice;
    }

    // method set dan get untuk procie, storage, memory, dan totalPrice
    function setProcie($procie) {
        $this->procie = $procie;
    }
    function setStorage($storage) {
        $this->storage = $storage;
    }
    function setMemory($memory) {
        $this->memory = $memory;
    }
    function setTotalPrice($totalPrice) {
        $this->totalPrice = $totalPrice;
    }

    function getProcie() {
        return $this->procie;
    }
    function getStorage() {
        return $this->storage;
    }
    function getMemory() {
        return $this->memory;
    }
    function getTotalPrice() {
        return $this->totalPrice;
    }

    // method print
    function printComputer() {
        $this->procie->printProcie();
        $this->storage->printDisk();
        $this->memory->printRam();
        echo "Total Price : ". "Rp. ". $this->getTotalPrice(). "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>